libcloud/test/compute/test_digitalocean_v2.py

```diff
@@ -518,6 +534,20 @@ def _v2_floating_ips_167_138_123_111(self, method, url, body, headers):
         else:
             raise NotImplementedError()
 
+    def _v2_floating_ips_133_166_122_204_actions(self, method, url, body, headers):
+        if method == 'POST':
+            body = self.fixtures.load('attach_floating_ip.json')
+            return (httplib.CREATED, body, {}, httplib.responses[httplib.CREATED])
+        else:
+            raise NotImplementedError()
+
+    def _v2_floating_ips_154_138_103_175_actions(self, method, url, body, headers):
+        if method == 'POST':
+            body = self.fixtures.load('detach_floating_ip.json')
+            return (httplib.CREATED, body, {}, httplib.responses[httplib.CREATED])
+        else:
+            raise NotImplementedError()
+
```

Comment:
Out of scope for this pull request, but I'm not a huge fan of this style because the raise path is never exercised and lowers the coverage. I think I'd prefer something like `assert method in ['GET', 'POST']` at the beginning of the function. Not sure it's worth the trouble, and for now consistency is better!